frame/support/src/weights.rs

```diff
@@ -315,6 +353,11 @@ impl sp_runtime::traits::Printable for PostDispatchInfo {
 		match self.actual_weight {
 			Some(weight) => weight.print(),
 			None => "max-weight".print(),
+		};
+		" pays_fee=".print();
+		match self.pays_fee {
+			Pays::Yes => "true".print(),
+			Pays::No => "false".print(),
```

Comment:
```suggestion
			Pays::Yes => "Yes".print(),
			Pays::No => "No".print(),
```
nit: have the enum consistent with the printing?